mews-api.md

```diff
@@ -517,31 +517,31 @@ This is example of a _successful_ response. In case an error occurred, the respo
 
 ### Process Group
 
-\[`async`\] This operation allows channel manager to push reservation group \(i.e. _booking_\) to Mews. This option allows creations, modifications and \(partial\) cancellations. Mews will process and confirm the booking asynchronously.
+\[`async`\] This operation allows the channel manager to push reservation groups \(i.e. _bookings_\) to Mews. This option allows creations, modifications and \(partial\) cancellations. Mews will process and confirm the booking asynchronously.
```

Comment:
Debatable, it allows to push just 1 group per call.

Reply:
Agree, will re-submit with:
a reservation group \(i.e. _booking_)\